Scripts/Runtime/Entities/Lifecycle/Migration/MigrationUtil.cs

```diff
@@ -0,0 +1,252 @@
+using Anvil.CSharp.Logging;
+using System;
+using Unity.Burst;
+using Unity.Collections;
+using Unity.Collections.LowLevel.Unsafe;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    /// <summary>
+    /// Helper class for when Entities are migrating from one <see cref="World"/> to another.
+    /// </summary>
+    public static class MigrationUtil
```

Comment:
Yep, I'm cool with that. Done.

Reply:
You're not going to like me but since they're all extension methods now `WorldEntityMigrationExtension` maybe makes more sense?

It's spring time and I really miss my 🚲 
(seriously though, you can leave it as is. I'm just splitting hairs)

Reply:
Cool we discussed and it's called `EntityWorldMigrationX` for the Extension, System and Observer now.